src/types.d.ts

```diff
@@ -178,6 +183,10 @@ declare global {
         info: string,
         warning: string
     }
+    interface Panel {
```

Comment:
We should rename this to `PanelConfig` because we have a component with the same name.

Reply:
yes, sounds good, I renamed it.